Specs/addDefaultMatchers.js

```diff
@@ -263,16 +263,16 @@ define([
 
             toPickPrimitive : function(util, customEqualityTesters) {
                 return {
-                    compare : function(actual, expected) {
-                        return pickPrimitiveEquals(actual, expected);
+                    compare : function(actual, expected, x, y, size) {
+                        return pickPrimitiveEquals(actual, expected, x, y, size);
                     }
                 };
             },
 
             notToPick : function(util, customEqualityTesters) {
                 return {
-                    compare : function(actual, expected) {
-                        return pickPrimitiveEquals(actual, undefined);
+                    compare : function(actual, expected, x, y, size) {
+                        return pickPrimitiveEquals(actual, undefined, x, y, size);
```

Comment:
For these two send in `width` and `height` instead of just `size` for consistency.